smithy-docgen/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+plugins {
```

Comment:
Could we also add a `README.md` matching the existing one ([here](https://github.com/smithy-lang/smithy-docgen))

Reply:
Yeah, good point, I meant to add that.